src/client/views/run.html

```diff
@@ -67,6 +67,7 @@ <h1 id="title" class="ms-progress-component__title ms-font-xxl">Script Lab</h1>
 					<div class="hostButton ms-BrandIcon--Icon48 ms-BrandIcon--powerpoint" data-host="POWERPOINT"></div>
 					<div class="hostButton ms-BrandIcon--Icon48 ms-BrandIcon--onenote" data-host="ONENOTE"></div>
 					<div class="hostButton ms-BrandIcon--Icon48 ms-BrandIcon--project" data-host="PROJECT"></div>
+					<div class="hostButton ms-BrandIcon--Icon48 ms-BrandIcon--outlook" data-host="OUTLOOK"></div>
 					<div class="hostButton ms-Icon ms-Icon--Group" data-host="TEAMS"></div>
```

Comment:
Sure.  I would remove OneNote as well, come to think of it.  @annich-MS ,  could you remove both?